README.md

```diff
@@ -34,8 +34,32 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
```

Comment:
go to this website for better understand of Semantic HTML
https://www.lifewire.com/why-use-semantic-html-3468271

Reply:
Yes, I answered the same meaning as what is on the article.